src/patterns/molecules/breadcrumb/breadcrumb.hbs

```diff
@@ -0,0 +1,21 @@
+---
+name: Breadcrumb
+description: A type of navigation menu used to provide a list of links based on current page within the site hierarchy, allowing users to visit previous pages from the current page.
+order: 1
+notes: |
+    - Use the `.mzp-t-dark` options to use dark mode.
+    - Use a `<ol>` to use semantic HTML and structure.
+    - Using the `nav` element is optional, but reccomended.
+---
+<nav aria-label="breadcrumb">
+  <ol class="mzp-c-breadcrumb">
```

Comment:
With the main component class moving to the `nav` I think this would become `mzp-c-breadcrumb-list`